### PR TITLE
Add intacctcoleenho-sanjose's list

### PR DESCRIPTION
**Pull request by intacctcoleenho**
Commit body added favorite restaurants.  Might need to add additional
content.

### CODE REVIEW

**Comment by awoodson13** [comment]
not sure how to see your list

**Comment by herecomesjaycee** [comment]
Hello! I cannot list the list as well.

**Comment by intacctcoleenho** [comment]
@awoodson13 @herecomesjaycee I'll see what I can do to make this work...

**Comment by intacctcoleenho** [comment]
Ok, content for file is now appearing...thanks for feedback!

**Comment by hli-ibm** [comment]
it looks fine

**Comment by herecomesjaycee** [comment]
Awesome work!

File: intacctcoleenho-sanjose.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# San Jose, CA
+
+### Places to Eat
+1. Poor House Bistro (NOLA food)
+1. San Pedro Market (different places together)
+1. Scott's Seafood
```
Comment (intacctcoleenho) [inline intacctcoleenho-sanjose.md:6]: I think I ate here once...

**Pull request merged**